sky/jobs/utils.py

```diff
@@ -855,8 +871,15 @@ def stream_logs(job_id: Optional[int],
         with open(controller_log_path, 'r', newline='', encoding='utf-8') as f:
             # Note: we do not need to care about start_stream_at here, since
             # that should be in the job log printed above.
-            for line in f:
-                print(line, end='')
+            if tail is not None and tail > 0:
```

Comment:
What does `tail <= 0` even mean... should we just assert `tail > 0` since we expect to use `None` to indicate no tail?